### DISCUSSION TITLE
v2 doesn't work, 'OpenAIAuth' is not a package

### DISCUSSION THREAD

**Discussion by K1yomi** [acheong08/ChatGPT#815] [Q&A] [answered]
Uninstall OpenAIAuth and update to the latest 0.3.2
But it still shows error：
ModuleNotFoundError: No module named 'OpenAIAuth.OpenAIAuth'; 'OpenAIAuth' is not a package

**Comment by acheong08** [comment, marked as answer]
V2 doesn't work anymore in general

**Reply by K1yomi** [reply]
ok, thx

**Comment by imconical** [comment]
I have the same question. Thanks.

**Comment by XingchenCedricXu** [comment]
Same here. I was also wondering if you could revise the example code to include the "plus" function.
